docs/how-to/how-to-build-and-test-ccv0.md

```diff
@@ -333,9 +333,14 @@ to create containers from these images using `crictl`:
 the guest agent code has been implemented, the Kata confidential-containers build needs to be run with 
 `export SKOPEO=yes` as documented [above](#basic-script-set-up-and-optional-environment-variables) in order to use the
 image signature verification function.
+
 - If you don't already have a Kata sandbox pod created with `crictl`, then follow the 
 [instructions above](#using-crictl-for-end-to-end-provisioning-of-a-kata-confidential-containers-pod-with-an-unencrypted-image)
 up to, and including, the `~/ccv0.sh crictl_create_cc_pod` command.
+
+- In order to enable the guest image, you will need to copy over the signature files needed by running 
```

Comment:
You might want to mention here that the `quay_policy.json` file is also copied, as in a early phrase it is mentioned a policy file is passed to the agent.